postgres/src/main/scala/zio/sql/postgresql/PostgresModule.scala

```diff
@@ -6,14 +6,9 @@ import zio.sql.Jdbc
  */
 trait PostgresModule extends Jdbc { self =>
 
-  object PostgresFunctionDef {
-    val Sind = FunctionDef[Double, Double](FunctionName("sind"))
-  }
-
-  override def renderRead(read: self.Read[_]): String = {
-    val builder = new StringBuilder
+  sealed case class Renderer(builder: StringBuilder = new StringBuilder) {
```

Comment:
Looks like `private final class` would be a better fit here than `sealed case class`.